skfp/model_selection/scaffold_split.py

```diff
@@ -0,0 +1,371 @@
+import warnings
+from collections import defaultdict
+from collections.abc import Sequence
+from numbers import Integral
+from typing import Any, Optional, Union
+
+from rdkit.Chem import Mol
+from rdkit.Chem.Scaffolds import MurckoScaffold
+from sklearn.utils._param_validation import Interval, RealNotInt, validate_params
+
+from skfp.model_selection.utils import (
+    ensure_nonempty_list,
+    get_data_from_indices,
+    split_additional_data,
+    validate_train_test_sizes,
+    validate_train_valid_test_split_sizes,
+)
+from skfp.utils.validators import ensure_mols
+
+
+@validate_params(
+    {
+        "data": ["array-like"],
+        "additional_data": ["tuple"],
```

Comment:
`tuple` does not work? If so, just use `Any`

Reply:
It does not work with either `Any` nor `tuple` for some reason